x-pack/plugins/alerting/server/application/maintenance_window/constants.ts

```diff
@@ -11,3 +11,10 @@ export const maintenanceWindowStatus = {
   FINISHED: 'finished',
   ARCHIVED: 'archived',
 } as const;
+
+export const maintenanceWindowCategoryIdTypes = {
```

Comment:
why do we need to create a new `const` and not re-use the one here => x-pack/plugins/alerting/common/routes/maintenance_window/shared/constants/v1.ts

Reply:
it's to avoid sharing types between the route and application layer.